.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+name: tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Build contracts
+      run: make all-via-docker
+
+    - name: Run tests for secp256r1_blake160 rust
+      run: cargo test
+      working-directory: ./tests/secp256r1_blake160
+
+    - name: Set up QEMU
+      run: |
+        sudo apt update
+        sudo apt install -y qemu-user qemu-user-binfmt
```

Comment:
Do we need qemu?

Reply:
The program `./build/secp256r1_blake160_c` (a simply program to test libecc functionailty) is a riscv binary. I used qemu to run that.

Reply:
always use ckb-vm to run it. You can choose `asm64`:
```
cargo install --example asm64 --branch rvv --git https://github.com/nervosnetwork/ckb-vm.git --features asm ckb-vm
```